test/runners/python_spec.js

```diff
@@ -322,5 +322,31 @@ describe('Fixed and new features', function() {
         done();
       });
     });
+
+    it(`should support unicode output (log) (Python${v} cw-2)`, function(done) {
+      runner.run({
+        language: 'python',
+        languageVersion: v,
+        testFramework: 'cw-2',
+        code: 'a = 1',
+        fixture: 'print("u\\uac00")',
```

Comment:
Did you mean `fixture: 'print(u"\\uac00")',`? `"u` vs `u"`. Not sure if that makes any difference.

Reply:
For Python 2.7, fixing this should make the test pass.
```python
print(u"\uac00") #=> &#44032;
print("u\uac00") #=> u\uac00
```